packages/components/src/spectrum/picker/PickerUtils.ts

```diff
@@ -120,11 +120,11 @@ export function isNormalizedPickerSection(
  * @param itemOrSection The picker item or section
  * @returns A `PickerItemKey` for the picker item
  */
-function normalizeItemKey(item: PickerItem): PickerItemKey;
-function normalizeItemKey(section: PickerSection): Key;
+function normalizeItemKey(item: PickerItem): PickerItemKey | undefined;
```

Comment:
I'd update the doc `@returns` as to when it returns `undefined`, as this is unexpected from the current docs/name.